tests/test_issue_fixes.py

```
@@ -0,0 +1,208 @@
+
+import json
+import tempfile
+from pathlib import Path
+
+import pytest
+
+
+# --- #70 -----------------------------------------------------------------
+
+
+def _extract_ts(source: str, filename: str = "test.ts"):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_extract_ts`

(ANN202)